invenio_indexer/api.py

```diff
@@ -69,27 +76,42 @@ def publish(self, data, **kwargs):
 
 
 class RecordIndexer(object):
-    """Record indexer.
-
-    Provides an interface for indexing records in Elasticsearch. Bulk indexing
-    works by queuing requests for indexing records and processing these
-    requests in bulk.
-
-    Elasticsearch index and doc_type for a record is determined from the
-    ``$schema`` attribute.
-
-    :param search_client: Elasticsearch client. Defaults to
-        ``current_search_client``
-    :param exchange: ``kombu.Exchange`` instance for message queue.
-    :param queue: ``kombu.Queue`` instance for message queue.
-    :param routing_key: Routing key for message queue.
-    :param version_type: Elasticsearch version type. Defaults to
-        ``external_gte``.
+    r"""Provide an interface for indexing records in Elasticsearch.
+
+    Bulk indexing works by queuing requests for indexing records and processing
+    these requests in bulk.
+
+    Example of bulk indexing:
+
+    >>> from flask import Flask
+    >>> from invenio_records.models import RecordMetadata
+    >>> from invenio_indexer.api import RecordIndexer
+
+    >>> # put them in a queue
```

Comment:
It can be normal text and not executed comment. You might want to join `app = Flask('myapp')` with the import statements and then say that the records are put in a queue.